### PR TITLE
Feature 54 implement window switch

### PR DESCRIPTION
**Pull request by ty-desj**
Added switch to window by title. Defaults to switch to next window if no title given. Raises exception if window not found.

### CODE REVIEW

File: src/tass/actions/selenium.py [hunk 1]
```diff
@@ -266,6 +266,42 @@ def switch_frame(driver, frame, find=_find_element):
...
+                return
+    elif (isinstance(title, str)):
+        for handle in driver.window_handles:
+            if (handle == cur_handle):
+                continue
+            else:
+                driver.switch_to.window(handle)
+                if (driver.title == title):
+                    return
+    raise ValueError('No window with title: {}'.format(title))
```
[the remark this answers is not in the document]
Comment (DominicParent) [reply src/tass/actions/selenium.py:302]: @ty-desj judging by the Python docs, ValueError seems fine. What do you not like about it?

[the remark this answers is not in the document]
Comment (ty-desj) [reply src/tass/actions/selenium.py:302]: I am thinking about for error handling purposes, whether we want to be able handle this error in a unique way or treat it like any other error.

[the remark this answers is not in the document]
Comment (ty-desj) [reply src/tass/actions/selenium.py:302]: I am comfortable leaving this as is for now, I can't come up with any reasonable reason that this particular error would need to be treated differently. Review at your convenience :smiley:

**Review by DominicParent — CHANGES_REQUESTED** [review]
Please resolve flake8 issues:
./src/tass/actions/selenium.py:271:1: W293 blank line contains whitespace
./src/tass/actions/selenium.py:274:62: W291 trailing whitespace
./src/tass/actions/selenium.py:276:1: W293 blank line contains whitespace
./src/tass/actions/selenium.py:283:67: W291 trailing whitespace
./src/tass/actions/selenium.py:285:1: W293 blank line contains whitespace

**Review by DominicParent — APPROVED** [review]

**Pull request merged**